networking_oneview/ml2/drivers/oneview/neutron_oneview_client.py

```diff
@@ -199,6 +199,12 @@ def _get_uplinksets_from_lig(self, network_type, lig_list):
         return uplinksets
 
     def _create_network_on_oneview(self, name, network_type, seg_id):
+        compatible_networks = self._verify_network_existence(name,
+                                                             network_type,
+                                                             seg_id)
+        if compatible_networks:
+            return compatible_networks[0]
```

Comment:
We cannot have two Networks with the same name.

Reply:
the get_all method can return a list with a unique network that fits with the past parameters, which is the case when the network inclusion on uplinksets was not successfully completed, but the network creation from Neutron' perspective was ok and the network keep existing in OneView, so that mapping was not created at the database and  the synchronization process kept trying to recreate the network at OneView.